### PR TITLE
[AIRFLOW-4681] Fix pylint problems for sensors module

### PR DESCRIPTION
**Pull request by mik-laj**
Make sure you have checked _all_ steps below.

### Jira

- [ ] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-4681\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-4681
  - In case you are fixing a typo in the documentation you can prepend your commit with \[AIRFLOW-XXX\], code changes always need a Jira issue.
  - In case you are proposing a fundamental code change, you need to create an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)).
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Description

- [ ] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [ ] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [ ] Passes `flake8`

### CODE REVIEW

**Comment by potiuk** [comment]
@mik-laj - there are some mypy/flake failures and name of the commit is wrong :)

File: airflow/sensors/hive_partition_sensor.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-
+"""Hive peratiton sensor module"""
```
Comment (BasPH) [inline airflow/sensors/hive_partition_sensor.py]:
```suggestion
"""Hive partition sensor module"""
```

File: airflow/sensors/hdfs_sensor.py [hunk 2]
```diff
@@ -45,6 +51,13 @@ def __init__(self,
                  *args,
                  **kwargs):
         super().__init__(*args, **kwargs)
+        if not snakebite_loaded:
+            raise ImportError(
+                'This HdfsSensor implementation requires snakebite, but '
+                'snakebite is not compatible with Python 3 '
+                '(as of August 2015). Please use Python 2 if you require '
+                'this hook  -- or help by submitting a PR!')
```
Comment (BasPH) [inline airflow/sensors/hdfs_sensor.py]: General question: I always wonder why there's code this short (75 chars) throughout the code base while the configured line length in Airflow is 110 characters. Is it enforced by some IDE default?

**Review by BasPH — CHANGES_REQUESTED** [review]
LGTM, besides one typo.

File: airflow/sensors/hdfs_sensor.py (hunk 2 above)
Comment (potiuk) [reply airflow/sensors/hdfs_sensor.py]: I think @mik-laj uses vim mostly ;). But indeed if it was IntelliJ I had to set line length to 110 in my IDE from the default 80 so I guess that's the reason.

**Review by potiuk — COMMENTED** [review]
Almost LGTM - just the string length and commit message :)

File: airflow/sensors/hdfs_sensor.py (hunk 2 above)
Comment (mik-laj) [reply airflow/sensors/hdfs_sensor.py]: This is a copy-paste from HDFS Hook:
https://github.com/apache/airflow/blob/master/airflow/hooks/hdfs_hook.py#L48-L52
I will correct it to take up to 110 characters.

**Comment by coopergillan** [comment]
Looks like this one just needs a rebase?

**Comment by mik-laj** [comment]
Too many conflicts. I will start from scratch.

**Pull request closed without merging**